src/main/java/seedu/address/logic/parser/ArgumentMultimap.java

```diff
@@ -52,6 +52,18 @@ public void put(Prefix prefix, String argValue) {
     }
 
     /**
+     * Returns multiple values of {@code prefix}.
+     */
+    public Optional<String> getMultipleValues(Prefix prefix) {
+        List<String> values = getAllValues(prefix);
+        String str = "";
+        for (String v : values) {
+            str += v + "\n";
+        }
+        return str.isEmpty() ? Optional.empty() : Optional.of(str);
+    }
+
+    /**
```

Comment:
How are we intending to display the numbers? Will it be:
Phone: 12345, 56789, 12321
or:
Phone: 12345
Phone: 56789
Phone: 12309
?

Reply:
Currently it will display as:
12345
56789

There is no "Phone: " as the original program does not display the phone numbers with "Phone: "